### DISCUSSION TITLE
direct link to a page

### DISCUSSION THREAD

**Discussion by aurel1sergent** [YetiForceCompany/YetiForceCRM#17599] [General]
Hi, we would like to acces a specific page from another domain : 
basically on https://domain.com/page insert a link to https://yetiforceurl/index.php?module=xxx&view=detail&record=nnn 

it seems the CSRF block it, is there a way to achieve it ? whitelist for the domain or any solution ?

**Comment by rskrzypczak** [comment]
here: https://doc.yetiforce.com/code/classes/Config-Security.html

**Reply by rskrzypczak** [reply]
probably loosen up `cookieSameSite` (Lax) and add domain to `allowedFormDomains`
